invenio_rdm_records/services/schemas/parent/access.py

```diff
@@ -9,6 +9,8 @@
 
 """Access schema for RDM parent record."""
 
+# TODO: Replace with invenio_records_resources.services.base.schema import *
```

Comment:
i.e. move the entire parent schema to `invenio-records-resources.base.schema`, also the schema for the secret links?
should the secret link code also be moved?

Reply:
Initially I was thinking just the parts that make sense - specifically first AgentSchema, but we likely need more as well that could be reused.